signup-service/src/main/java/io/mosip/signup/services/WebSocketHandler.java

```diff
@@ -55,12 +57,18 @@ public class WebSocketHandler {
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private IdentityVerificationRequestValidator identityVerificationRequestValidator;
```

Comment:
As this will not be reused, this can be a method in the same WebSocket Handler class.